functions/src/types/UserData.ts

```diff
@@ -1,7 +1,11 @@
 import * as admin from 'firebase-admin'
 import Timestamp = admin.firestore.Timestamp
+import { UID } from './uid'
 
 export interface UserData {
+    id: UID
+    subscriptionActive: boolean
+    quotaReached: boolean
```

Comment:
as a general rule, I think it would be clearer to use `is{variableName}` styling for `Boolean` values